lms/djangoapps/ccx/migrations/0002_add_master_course_staff_in_ccx.py

```diff
@@ -15,6 +14,7 @@ def add_master_course_staff_to_ccx_for_existing_ccx(apps, schema_editor):
     """
     Add all staff and admin of master course to respective CCX(s).
     """
+    CustomCourseForEdX = apps.get_model("ccx", "CustomCourseForEdX")
```

Comment:
@giocalitri DO we need to add ``if statement`` to check if it returned model or not?